prog/vnet/nic_nexus.rb

```diff
@@ -26,6 +26,13 @@ def before_run
     end
   end
 
+  label def wait_allocation
+    if nic.vm.allocated_at
```

Comment:
I assume no nic would be in this state without a vm

Reply:
yes